test/e2e/page-objects/pages/send/solana-confirm-tx-page.ts

```diff
@@ -71,6 +76,15 @@ class ConfirmSolanaTxPage {
     );
   }
 
+  /**
+   * Clicks the confirm button on the Solana transaction confirmation page
+   */
+  async clickOnConfirm(): Promise<void> {
+    await this.driver.waitForSelector(this.confirmButton);
+    await this.driver.clickElement(this.confirmButton, 3);
```

Comment:
we already have 3 retries as the default number in the `clickElement` method. So we can remove this

```suggestion
    await this.driver.clickElement(this.confirmButton);
```

Reply:
Updated